tensorboard/webapp/widgets/data_table/types.ts

```diff
@@ -80,6 +80,7 @@ export declare interface ColumnHeader {
   name: string;
   displayName: string;
   enabled: boolean;
+  tags?: string[];
```

Comment:
Why have you opted to display `differs` as a tag as opposed to a discrete value?

Reply:
I've realized after reading through this again that this is clearly a solution to display hparam related information in the column selector.

What I was trying to ask was why the data was structured as 
```
{
   enabled: string;
   displayName: string;
   ...,
   tags: ['differs'],
}
```

rather than
```
{
   enabled: string;
   displayName: string;
   ...,
   differs: true,
}
```

Reply:
My reasoning, mainly: The 'differs' value is really only relevant to hparams. We don't generally care to highlight this property about columns. 'tags' seemed like a generic-enough feature to allow me to add 'differs' for hparams without polluting the column selector api.

Secondarily: I hope that 'tags' would be useful for other use-cases in the future. (but I'm not necessarily counting on it).

Happy to reconsider if you think 'differs' should be part of the column selector api.